kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/client/DefaultJsonHandlerSuite.scala

```diff
@@ -458,6 +458,61 @@ class DefaultJsonHandlerSuite extends AnyFunSuite with TestUtils with VectorTest
     checkAnswer(actResult, expResult)
   }
 
+  test("write rows as json") {
```

Comment:
Is there any way to test by reading with Spark? Or is it just much easier to do this once we are actually writing tables?

Reply:
It will be easier once we have end-2-end. We did add the related infra test utils as part of the Parquet writer PRs.